### PR TITLE
Mobile-Unit-Tests —> Mobile

### PR DESCRIPTION
**Pull request by GGLozanov**
First iteration of Mobile unit test coverage.
Requested review.

### CODE REVIEW

**Review by msimov02 — APPROVED** [review]
LGTM!

**Review by marianbitsov23 — APPROVED** [review]
LGTM!

File: Mobile/tues_pairs/lib/modules/student.dart [hunk 1]
```diff
@@ -10,10 +7,10 @@ class Student extends User {
     double GPA, bool isTeacher,
     String username, String matchedUserID,
     List<String> skippedUserIDs}) :
-        super(
+      assert(isTeacher == false || isTeacher == null),
```
Comment (shristov-elsys) [inline Mobile/tues_pairs/lib/modules/student.dart:10]: Asserts are never used outside tests. As I see this is not  test

File: Mobile/tues_pairs/lib/modules/teacher.dart [hunk 2]
```diff
@@ -9,6 +9,7 @@ class Teacher extends User {
     String uid, String email,
     String photoURL, bool isTeacher,
     String username, String matchedUserID, List<String> skippedUserIDs}) :
+        assert(isTeacher == true || isTeacher == null),
```
Comment (shristov-elsys) [inline Mobile/tues_pairs/lib/modules/teacher.dart:12]: same here

**Review by shristov-elsys — COMMENTED** [review]
I would in general split this code review in several smaller ones

**Pull request merged**